Add MultiApps Documentation to VTB (#102)
idaholab/virtual_test_bed: doc/content/resources/multiapps_chps/chp_3_applications.md

### MultiApp Reactor Examples on VTB

The examples available on the Virtual Test Bed site adopt a number of different coupling approaches based on the specific reactor type and phenomena of focused interest. We briefly summarize some examples to show readers the flexibility in using MultiApps (see [vtb_examples]).

!media media/resources/vtb_examples.png
      style=display: block;margin-left:auto;margin-right:auto;width:100%;
      id=vtb_examples
      caption=Hierarchies of MultiApps for VTB examples.

> #### VTB-MSFR

Reviewer: for msfr, htgr, sfr and pb-fhr we have more than one model.
- let's link to the model in particular rather than the index page
- let's be more precise when referring to the model. For example instead of MSFR example, say MSFR core multiphysics simulation

#### VTB-HTGR

A Pebble-Bed Microreactor model is available on the VTB using a three-level MultiApps structure. Griffin is the parent application governing neutronics simulations, while Pronghorn solves a homogenized porous media flow problem. A CentroidMultiApp is then used to calculate the representative temperature profile within pebbles and TRISOs. This is an example of using MultiApps to handle similar physical phenomena (thermal heat transfer here) at different space scales. 

#### VTB-HP-MR

The Heat-Pipe Microreactor ([HP-MR](mrad/index.md)) uses a two-level MultiApps approach with a BISON parent application to govern the thermal physics within all the solid reactor components except for heat pipes, and a Sockeye child app to deal with heat pipe performance. A placeholder is added for future adoption of Griffin as parent app for neutronics simulation. When Griffin is integrated as parent, BISON and Sockeye will become child and grandchild applications, respectively.

#### VTB-PBFHR

> The Pebble Bed Fluoride-Salt-Cooled High-Temperature Reactor ([PB-FHR](pbfhr/index.md)) uses an approach similar to the HTGR example. Unlike the MSFR model in VTB where fuel is dissolved in the salt, the fuel of PBFHR has solid form so that delayed neutron information no longer needs to be handled by fluid dynamics application. However, the thermal physics inside the fuel pebbles need to be simulated by an additional grandchild Heat Conduction application.

Reviewer: ```suggestion
The Pebble Bed Fluoride-Salt-Cooled High-Temperature Reactor ([PB-FHR](pbfhr/index.md)) uses an approach similar to the HTGR example. Unlike the MSFR model in VTB where fuel is dissolved in the salt, the fuel of PBFHR has a solid form. Therefore delayed neutron information no longer needs to be handled by fluid dynamics application. However, the thermal physics inside the fuel pebbles need to be simulated by an additional grandchild Heat Conduction application.
```

#### VTB-SFR

The Sodium-cooled Fast Reactor ([SFR](sfr/sfr.md)) example presently has the most complex MultiApp hierarchy. As explained earlier, neutronics (Griffin) is naturally the parent application. The radial thermal expansion is simulated by a child application using the Tensor Mechanics module on the support plate with a preset inlet coolant temperature. The axial expansion as well as the axial temperature profile is simulated by a BISON child application to capture fuel behavior. The fuel temperature boundary condition is calculated by a grandchild application using SAM's coolant channel models. A special approach used in this example is that the axial expansion of the fuel is simulated by another grandchild BISON (i.e. two BISON simulations/MultiApps are involved) application instead of fully coupled with fuel thermal physics.

## Other Applications

The VTB examples generally employ MultiApps to couple different physical phenomena within a nuclear reactor system. Aside from multiphysics coupling, the MultiApps system can also be used in other scenarios.

### Multi-Spacescale

> The HTGR example uses MultiApps for non-multiphysics coupling. As mentioned early in this chapter, MultiApps are used in this example to connect a similar physical phenomenon (i.e., heat conduction) at two different space scales. In the HTGR example, the macroscopic reactor components involved in neutronics and thermal hydraulics simulations such as fuel and coolant are meter level, while single TRISO particles with fuel pebble are only millimeter level. Due to the complex multilayered structure of a TRISO particle, the particle needs to be separately simulated to capture its internal maximum temperatures. It is impractical to fully couple them as that would require an ultra fine mesh to capture thousands of TRISO particles. Instead, using a CentroidMultiApp, a virtual representative TRISO particle is simulated for each macroscopic fuel mesh element, providing representative local maximum fuel temperature information. This example shows the advantage of using MultiApps to study similar (or dissimilar) physical phenomena happening at very different space scale. Using similar approaches, it is possible to connect engineering scale simulations with mesoscale or even microscale physics.

Reviewer: link to the example. 
there are 3 inputs now for HTGR, 2 have multiapp setups

### MultiApp Reactor Examples on VTB

The examples available on the Virtual Test Bed site adopt a number of different coupling approaches based on the specific reactor type and phenomena of focused interest. We briefly summarize some examples to show readers the flexibility in using MultiApps (see [vtb_examples]).

!media media/resources/vtb_examples.png
      style=display: block;margin-left:auto;margin-right:auto;width:100%;
      id=vtb_examples
      caption=Hierarchies of MultiApps for VTB examples.

#### VTB-MSFR

The Molten Salt Fast Reactor ([MSFR](msfr/index.md)) example adopts a two-level MultiApps structure with Griffin as the parent app and Pronghorn as the child app. The parent app governs neutronics simulation, providing power distribution to the child app. The child app handles fluid dynamics simulation of molten salt fuels, providing temperature information as feedback to the parent app. MSFR is a special class of reactors as the molten salt is both fuel and coolant. As the molten salt flows out of the core region to transfer heat, the delayed neutron information is also calculated to transferred back to Griffin.

#### VTB-HTGR

> In the High-Temperature Gas-cooled Reactor ([HTGR](htgr/index.md)) category, a Pebble-Bed Microreactor ([PB-MR](pbmr/index.md))) model is available. Griffin is the parent application governing neutronics simulations, and MOOSE Heat Conduction and Thermal Hydraulics modules child applications calculate the heat transfer in solid (matrix + fuel) and fluid (coolant) regions, respectively. A special approach used in this example is the use of a CentroidMultiApp as the grandchild app to calculate the representative temperature profile within single TRISO particles using Heat Conduction Modules. This is an example of using MultiApps to handle similar physical phenomena (thermal here) at different space scales.

Reviewer: ```suggestion
A Pebble-Bed Microreactor model is available on the VTB using a three-level MultiApps structure. Griffin is the parent application governing neutronics simulations, while Pronghorn solves a homogenized porous media flow problem. A CentroidMultiApp is then used to calculate the representative temperature profile within pebbles and TRISOs. This is an example of using MultiApps to handle similar physical phenomena (thermal heat transfer here) at different space scales. 
```

both child input files use Pronghorn not the modules. There's a lot of modules code in them but you cant run them without prognhorn because of the matprops
